wasm-engines/docker/wasmi.Dockerfile

```diff
@@ -1,9 +1,4 @@
-FROM jwasinger/llvm-10:1.0
+FROM jwasinger/bench-base:1.0
 
-# install rust
-RUN curl https://sh.rustup.rs -sSf | \
-    sh -s -- --default-toolchain 1.42.0 -y && . $HOME/.cargo/env
-ENV PATH=/root/.cargo/bin:$PATH
-
-RUN git clone --single-branch --branch bench-time https://github.com/ewasm-benchmarking/wasmi.git --recursive
-RUN cd wasmi && cargo test --release
+RUN git clone --single-branch --branch bench-time https://github.com/ewasm-benchmarking/wasmi.git --recursive && \
```

Comment:
As mentioned the previous comment this indeed requires `cargo test --release` unless you fix the repo to include the bench code in `src/bin/instantiate.rs` (see my previous comment).